includes/components/class-admin.php

```diff
@@ -988,9 +988,10 @@ public function add_post_states( $states, $post ) {
 	 * Gets meta boxes.
 	 *
 	 * @param string $screen Screen name.
+	 * @param mixed  $category_id Category ID.
 	 * @return array
 	 */
-	protected function get_meta_boxes( $screen ) {
+	protected function get_meta_boxes( $screen, $category_id = null ) {
```

Comment:
This function will possibly have less changes after resolving the comments blow regarding the 2nd argument and saving the category ID in meta